shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profilePreferences/domain/User.kt

```diff
@@ -0,0 +1,24 @@
+package com.vml.tutorial.plantshop.profilePreferences.domain
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class User (
+    val cId: String,
+    var firstName: String?,
+    var lastName: String?,
+    val email: String,
+    var birthDate: String?,
+    var phoneNumber: String?,
+    var address: Address
```

Comment:
I also leave a comment here, can you make this (address) nullable?

Reply:
Done